http/oidc/src/test/java/org/wildfly/security/http/oidc/OidcBaseTest.java

```diff
@@ -215,4 +231,16 @@ protected String getCookieString(HttpServerCookie cookie) {
         return header.toString();
     }
 
+    protected void checkForScopeClaims(Callback callback) throws InvalidJwtException {
+        Credential credential = ((IdentityCredentialCallback)callback).getCredential();
+        String token = ((BearerTokenCredential) credential).getToken();
+        JwtClaims jwtClaims = new JwtConsumerBuilder().setSkipSignatureVerification().setSkipAllValidators().build().processToClaims(token);
+        String scopes = jwtClaims.getClaimValueAsString("scope");
```

Comment:
@PrarthonaPaul Just a minor, _getClaimValueAsString_ method can return null so it's best to put a nullcheck below

Reply:
Fixed!

Reply:
If the returned `scopes` is `null`, should the test fail?